sqlc.yaml

```
@@ -0,0 +1,8 @@
+version: "2"
+sql:
+  - engine: "postgresql"
+    schema: "schema.sql"
+    gen:
```

```suggestion
    queries: "query.sql"
    gen:
```
https://docs.sqlc.dev/en/stable/tutorials/getting-started-postgresql.html